packages/react/src/components/Fabric/Fabric.styles.ts

```diff
@@ -22,11 +20,6 @@ export const getStyles = (props: IFabricStyleProps): IFabricStyles => {
       theme.fonts.medium,
       {
         color: theme.palette.neutralPrimary,
```

Comment:
I like this change because, as you pointed out, it's a nice performance win and Fluent probably shouldn't be providing these styles to non-Fluent controls. However, I think there's a case to be made that this is a breaking change as any users relying on the behavior may not potentially have their designs/layout broken after upgrading to this version of Fluent.

We could add a new prop that allows us to conditionally remove these styles similar to how we approached style changes to `Stack`.

Reply:
Yeah, I thought about that, the only interesting part there would be what to do with the `Fabric` components used inside of our other components, although I guess we could expose the prop there in some way as well. I'll work on the change today.